app/helpers/format_content_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module FormatContentHelper
```

Comment:
I wonder if this is a good use case for a view component?

`FormattedContentCompent.new(content: content)` ...

In theory any time we find ourselves reaching for view helpers we should be using view components instead 🤔

Reply:
Want to keep consistent with bops-applicant so will have to add the view_component gem there too.

Reckon that's not overkill?
I guess we may want to move to that view approach on that app too

Reply:
I am in favour of view components in All Of The Places...but possibly I'll get over it 😆